build.gradle.kts

```diff
@@ -26,14 +26,17 @@ tasks.withType<KotlinCompile> {
 }
 
 tasks.withType<PatchPluginXmlTask> {
-    sinceBuild.set("222")
+    sinceBuild.set("232")
 }
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-    version.set("2022.2")
+    pluginsRepositories {
+        custom("https://plugins.jetbrains.com/plugins/eap/ideavim")
+    }
+    version.set("2023.3.2")
     updateSinceUntilBuild.set(false)
-    plugins.set(listOf("IdeaVIM:1.11.1"))
+    plugins.set(listOf("IdeaVIM:2.8.0-eap.4"))
```

Comment:
Lets bump the dependency to the released version `2.8.0` and get this done :slightly_smiling_face: